frontend/src/components/course-search/CourseSearch.tsx

```diff
@@ -278,3 +337,59 @@ const CourseSearchRow = memo(function CourseSearchRow(props: {
         </div>
     );
 });
+
+const CourseSearchEnd = memo(function CourseSearchEnd(props: { text: string }) {
+    return <div className={Css.end}>({props.text})</div>;
+});
+
+const HistoricalSearchResults = memo(function HistoricalSearch(props: {
+    sections: APIv4.Section[] | undefined;
+}) {
+    const activeTerm = useUserStore((user) => user.activeTerm);
+    const setPopup = useStore((store) => store.setPopup);
+
+    const sections = props.sections?.filter((section) => {
+        return !(
+            section.identifier.term === activeTerm.term &&
+            section.identifier.year === activeTerm.year
+        );
+    });
+
+    if (sections === undefined) {
+        return (
+            <div className={Css.emptyHistoricalSearchResults}>
+                (loading courses from other terms...)
+            </div>
+        );
+    }
+    if (sections.length === 0) {
+        return (
+            <div className={Css.emptyHistoricalSearchResults}>
+                (no historical records of courses found)
+            </div>
+        );
+    }
+
+    return (
+        <div className={Css.historicalSearchResults}>
+            <h4>Are you looking for these sections from recent terms?</h4>
+            <div>Click on any section to see its details!</div>
```

Comment:
We don't need to say things like this. For every section ever listed anywhere, clicking on the section always shows details. So this is not a special behavior and we shouldn't need to point it out like this. Also, instead of "Are you looking for these sections from recent terms?" we can just say "found results from previous terms" or something similar.